packages/atlas/src/components/EmojiWrapper/EmojiWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from '@emotion/styled'
+import React, { forwardRef, useEffect, useRef } from 'react'
+import mergeRefs from 'react-merge-refs'
+import twemoji from 'twemoji'
+
+export type EmojiWrapperProps = {
+  // block prop should be used if the emoji is used outside of text and should behave like block element
+  block?: boolean
+  className?: string
+  children?: React.ReactNode
+}
+
+export const EmojiWrapper = forwardRef<HTMLSpanElement, EmojiWrapperProps>(
+  ({ block, className, children }, outsideRef) => {
+    const ref = useRef<HTMLDivElement>(null)
+
+    // parse the DOM node and replace all emojis with twemojis
+    // no deps array, run on every render
+    useEffect(() => {
```

Comment:
So, basically for every node where we want to see twemojis, we would need to apply this EmojiiWrapper? 
We can't just wrap `<App>` in EmojiWrapper and see twemojis in the entire app?